Style.md

```diff
@@ -733,5 +767,12 @@
 ### Callback
 
   - First argument must always be `err`
+  - Inline callbacks should use arrow functions
   - If a function takes a `callback` argument, it **must** be called on `process.nextTick()`. Otherwise, the argument name **must** be `next` to clearly declare that it may get called on same tick
   - Callbacks should always be called with explicit `return`
+
+### Promises
+
+  - Public interfaces should (not must) return a promise when no callback is provided
+  - Promises should not be used internally
+  - Only native promises are allowed. Third party promise implementations are not allowed
```

Comment:
This one was given to me by @hueniverse, so he'll have to elaborate. If it were up to me, there would be no promises :-)